cmake/version.h.in

```diff
@@ -0,0 +1,29 @@
+#ifndef SKA_ASTROACCELERATE_VERSION_H
+#define SKA_ASTROACCELERATE_VERSION_H
+
+
+#define ASTROACCELERATE_BUILD_TYPE "@CMAKE_BUILD_TYPE@"
+#define ASTROACCELERATE_VERSION "@PROJECT_VERSION@"
+#define PROJECT_NAME "@PROJECT_NAME@"
+#define ASTROACCELERATE_RCS_VERSION "@PROJECT_SOURCE_VERSION@"
+#define ASTROACCELERATE_CUDA_MAJOR_VERSION @CUDA_VERSION_MAJOR@
+#define ASTROACCELERATE_CUDA_MINOR_VERSION @CUDA_VERSION_MINOR@
+#define ASTROACCELERATE_CUDA_VERSION @CUDA_VERSION_STRING@
+
```

Comment:
Can we add the architecture version here?

Reply:
Arent we doing fat bins? If that is the case, architecture would be irrelevant.

Reply:
As above, could you put the same convention here? `ASTRO_ACCELERATE_`